src/BloomFilter.cpp

```diff
@@ -18,8 +18,43 @@
 #include <fstream>
 #include <cmath>
 #include <assert.h>
+#include <sys/stat.h>
 #include "BloomFilter.hpp"
 
+
+#define HDR_MAGIC1 0xBF
+#define HDR_MAGIC2 0xAA
+#define HDR_MAGIC3 0xFE
+#define HDR_MAGIC4 0xED
+
+#define VERSION_MAJOR 1
+#define VERSION_MINOR 0
+
+#define INLINE inline
+
+
+// File Headers
+typedef struct LegacyHeader_s {
```

Comment:
Legacy support isn't needed, our library and backend endpoint will be versioned. Let's keep the code simple and remove any legacy support or other references to the old bloom filter.